dataactcore/utils/requestDictionary.py

```diff
@@ -46,9 +46,8 @@ def derive(request, optional_request=False):
                 request_data = request.form.to_dict()
                 request_data['_files'] = request.files
                 for key, value in request_data.items():
-                    if value == 'null':
+                    if value in ('null', 'None'):
```

Comment:
Could we do a `value.lower().strip() in ('null', 'none')` to account for bad values ('NULL', 'none')